rules/windows/builtin/security/win_security_successful_external_remote_rdp_login.yml

```
@@ -23,15 +24,21 @@ detection:
     selection:
         EventID: 4624
         LogonType: 10
-    filter:
-        SubjectUserName: '-'
+    filter_ipv4:
         IpAddress|cidr:
-            - 10.0.0.0/8
-            - 172.16.0.0/12
-            - 192.168.0.0/16
-            - 224.0.0.0/4
-            - 127.0.0.0/8
-    condition: selection and not filter
+            - '127.0.0.0/8'
+            - '10.0.0.0/8'
+            - '172.16.0.0/12'
+            - '192.168.0.0/16'
+    filter_ipv6:
+        - IpAddress: '::1' # IPv6 loopback
+        - IpAddress|startswith:
+            - 'fe80:'  # link-local address
+            - 'fc00:'  # private address range fc00::/7
+            - 'fd00:'  # private address range fd00::/8
```

```suggestion
            - 'fc'  # private address range fc00::/7
            - 'fd'  # private address range fc00::/7
```
see comment of the other rule